code/datums/map_config.dm

```diff
@@ -149,15 +149,15 @@
 		log_world("map_config space_empty_levels is not a number!")
 		return
 
-	if(!("mine_disable" in json)) //Bypass mineload so we don't load any mining vessels period.
+	if("mine_disable" in json)
+		mine_disable = json["mine_disable"]
+	if(!("mine_disable" in json) || !mine_disable) //Bypass mineload so we don't load any mining vessels period.
```

Comment:
Wait does if("mine_disable" in json) mean if the variable is present, and not the value of said variable?
Meaning we were never actually checking if it was set to 0 or 1, so by setting the default to 1 all attempts to make it 0 again in the json also failed?

Reply:
Good point